documentation/src/main/java/outgoings/SplitterMultiExample.java

```diff
@@ -0,0 +1,37 @@
+package outgoings;
+
+import java.util.Objects;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.operators.multi.split.MultiSplitter;
+
+public class SplitterMultiExample {
+
+    // <code>
+    enum Caps {
+        ALL_CAPS,
+        ALL_LOW,
+        MIXED
+    }
+
+    @Incoming("in")
+    @Outgoing("sink1")
+    @Outgoing("sink2")
+    @Outgoing("sink3")
+    public MultiSplitter<String, Caps> reshape(Multi<String> in) {
```

Comment:
I don't see how the caps are associated with the outgoings?

Reply:
It's by order of `@Outgoing` declarations and enum ordinals.
We can also have a ignore case matching between enum names and channel names and fallback to the order matching. WDYT? 

 I'll add a line to the doc.

Reply:
yes, names and ignoring the case would work fine.